editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -103,6 +102,8 @@ const FailJestOnCanvasError = () => {
   const stableCallback = React.useCallback((newRuntimeErrors: Array<RuntimeErrorInfo>) => {
     // we have new runtime errors, let's take the tests down
     if (newRuntimeErrors.length > 0) {
+      console.error('Canvas Error!!!!!', newRuntimeErrors[0]?.error)
```

Comment:
Hahahaha! Is this needed or is the `fail` enough?

Reply:
it's needed 🤦 
this way it's 100% certain that the CI output contains a readable stack trace and error info